woocommerce/woocommerce
# Fix for free shipping handling of minimum order rule

### Changes proposed in this Pull Request:





Fixes the calculation from when `Apply minimum order rule before coupon discount` is active and prices are entered inclusive of tax and displayed that way in cart and checkout, free shipping is no longer available (based on the minimum order amount including tax), as it uses the pre-tax amount when a coupon is applied.

Closes #28752 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to **WooCommerce > Settings > Tax**
2. Set **Prices entered with tax** to **Yes, I will enter prices inclusive of tax**
3. Set **Display prices during cart and checkout** to **Including tax**
4. Add a tax rate
5. Add Free shipping with a required minimum order amount
6. Check the **Apply minimum order rule before coupon discount** option for Free shipping
7. Add a physical product with a price inclusive of tax is above the minimum order amount, but the pre-tax amount is below the minimum order amount
8. Add a coupon that doesn't include free shipping
9. Add the product to the cart
10. Confirm free shipping is available
11. Add the coupon
12. Confirm free shipping is still available
